src/lightning/pytorch/cli.py

```diff
@@ -293,6 +296,23 @@ def save_config(self, trainer, pl_module, stage):
             instead.
 
         """
+        if isinstance(trainer.logger, WandbLogger):
```

Comment:
The code below looks good in general.
However, this `save_config` hook is meant to be overridden by users who want to add their own saving logic. We should not implement it here. I propose to simply create a new callback:

```py
class LoggerSaveConfigCallback(SaveConfigCallback)
    def save_config(...):
        # your logic here
```
        
and then we register that one.

What do you think?